src/app/profile/receipts/receipts.tpl.html

```diff
@@ -0,0 +1,80 @@
+<div class="screenContent" tabindex="-1" role="dialog">
+  <div class="container">
+    <div class="col-md-12">
+      <div ng-if="$ctrl.retrievingError" class="alert alert-danger" role="alert">
+        <span>{{$ctrl.retrievingError}}</span>
+      </div>
+      <div class="panel panel-margin-bottom">
+        <div class="panel-body">
+          <div class="row">
+            <div class="col-md-12">
+              <h3>Receipts</h3>
+
+              <form ng-if="$ctrl.receipts && $ctrl.receipts.length" class="form-inline mt">
+                <div class="form-group">
+                  <span class="pr-">View My Gifts:</span>
+                </div>
+                <div class="input-group">
+                  <div class="btn-group" uib-dropdown>
+                    <button ng-click="$ctrl.toggle = !$ctrl.toggle" class="btn btn-default dropdown-toggle" uib-dropdown-toggle>
+                      <span class="hidden-xs">{{$ctrl.showYear}} <span class="caret"></span></span>
+                      <span class="visible-xs"><span class="fa fa-filter"></span></span>
+                    </button>
+                    <ul ng-click="$ctrl.toggle = false" class="dropdown-menu" uib-dropdown-menu>
```

Comment:
This should be a list of the last 10 years and have $ctrl.setYear() drive the loading of the receipts from the API for that year.